code42/py42
# Upgrade web restore API usage to v9

### Description of Change ###

Upgrades the endpoint for starting a web restore to use v9. This mirrors the console UI. It is in our developers' best interest to mirror the console UI whenever possible such that debugging requests can be much easier.

### Issues Resolved ###
N/a


### Testing Procedure ###
Verify you can still restore files
Verify you can still restore multiple files
Verify you can cancel a restore job after starting it.

### PR Checklist ###
Did you remember to do the below?

- [n/a] Add an entry to CHANGELOG.md describing this change
- [n/a] Add docstrings for any new public parameters / methods / classes
